docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_ANDROID.mdx

```diff
@@ -29,12 +29,13 @@ For reference see the [CLI's docs](https://github.com/mateusz1913/vision-camera-
 
 2. Register the package in MainApplication.java
 
-```java {6}
```

Comment:
This is discouraged way of highlighting content in docusaurus

Reply:
ahh nice!